.moban.dt/core-setup.py.jj2

```diff
@@ -0,0 +1,41 @@
+{% extends 'coala-setup.py.jj2'%}
```

Comment:
I don't have much idea about this format but there is a space before `extends`, is it required?

Reply:
Is there are a style guide for jinja2? :)

We have a bear, and I have enabled it.

Does it need enhancing? :P

Reply:
Don't know. Found this https://github.com/pallets/jinja#nutshell 
> Is there a style guide for jinja2? :)

Reply:
I've added spaces near the `{%` and `%}`.